src/test/scala/org/kibanaLoadTest/scenario/Dashboard.scala

```diff
@@ -3,132 +3,166 @@ package org.kibanaLoadTest.scenario
+        ).exec(
           http("query visualizations")
             .post("/api/saved_objects/_bulk_get")
-            .body(
-              StringBody(
-                "[" +
-                  "${vizListString}"
-                    .concat(
-                      ", { \"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"  }]"
-                    )
-              )
-            )
+            .body(StringBody("[${vizListString}, {\"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"}]"))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
           http("query search & map")
             .post("/api/saved_objects/_bulk_get")
             .body(StringBody("""[ ${searchAndMapString} ]""".stripMargin))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
+          http("query index pattern meta fields")
+          .get("/api/index_patterns/_fields_for_wildcard")
+            .queryParam("pattern", "kibana_sample_data_ecommerce")
+            .queryParam("meta_fields", "_source")
+            .queryParam("meta_fields", "_id")
+            .queryParam("meta_fields", "_type")
+            .queryParam("meta_fields", "_index")
+            .queryParam("meta_fields", "_score")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
+          http("query index pattern search fields")
+            .get("/api/saved_objects/_find")
+            .queryParam("fields", "title")
+            .queryParam("per_page", "10")
+            .queryParam("search", "kibana_sample_data_ecommerce")
+            .queryParam("search_fields", "title")
+            .queryParam("type", "index-pattern")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
+          http("query input control settings")
+            .get("/api/input_control_vis/settings")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
           http("query timeseries data")
             .post("/api/metrics/vis/data")
             .body(ElFileBody("data/timeSeriesPayload.json"))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
           http("query gauge data")
             .post("/api/metrics/vis/data")
             .body(ElFileBody("data/gaugePayload.json"))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
```

Comment:
`api/metrics/vis/data` requests are duplicated (I checked that body is exactly the same)

![image](https://user-images.githubusercontent.com/10977896/114020978-76185f80-9870-11eb-8ba0-70a82c73c8a2.png)

Is there any reason to make it twice? I'm not sure I should repeat it on load scenario